### DISCUSSION TITLE
help and feedback wanted on implementing streaming support into the sonification library.

### DISCUSSION THREAD

**Discussion by venkateshpotluri** [make4all/psst#43] [Announcements]
More details in PR #41 . posting hear to keep everybody informed. Please add your feedback to the PR directly as having the code and comments at one place will make it easy.

Also if I remember correctly, @jrthompson33  we wanted to sync up this week regarding streaming data. I think having this would give us a good base to discuss. Would you like to meet on Wednesday (preferably evening), or sync up after the break?

**Comment by jrthompson33** [comment]
Sorry @venkateshpotluri, I didn't have enough time this week to work on the streaming testbench. Let's plan to sync after the break. I'll post a PR with the example code before we sync up.
